charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+helm:
```

Comment:
why helm and not super-agent?

Reply:
The way I see if a values object makes sense or is redundant is to ask myself what would the values look like if we set an umbrella chart. Let's create an example. Imagine this chart goes inside the `nri-bundle`. The values on the bundle with your suggestion would look like this:
```yaml
super-agent:
  super-agent:
    create: true
```

On the other hand, with `helm` it would look like:
```yaml
super-agent:
  helm:
    create: true
```
With this example, I can say that the object `helm` is not the agent but the helm release, and it makes sense to call it `helm` :)

Reply:
Really nice explanation 😃 

I guess having it "at level 0" (without `helm:`) it does not make sense. Then I agree that helm makes sense!